lib/functions.php

```diff
@@ -2527,6 +2541,14 @@ function password_check($cryptedpassword,$plainpassword,$attribute='userpassword
 
 			break;
 
+		# Argon2 crypted passwords
+		case 'argon2':
+			if (password_verify($plainpassword, $cryptedpassword))
+				return true;
```

Comment:
Using one-liner is more readable `return password_verify($plainpassword, $cryptedpassword);`

Reply:
> Using one-liner is more readable `return password_verify($plainpassword, $cryptedpassword);`

Yes, you're rights. I write it like this to match with the style of this function :

https://github.com/leenooks/phpLDAPadmin/blob/master/lib/functions.php#L2492

I could also provide a commit to fix it in all the function, say me.